app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown.erb

```diff
@@ -0,0 +1,35 @@
+<li class='navbar-account_item navbar-account_item-cart' id='mini-cart' data-turbolinks-temporary>
+  <%= link_to cart_url, class: "nav-icon-stack hidden-grid-float-breakpoint" do %>
+    <i class='fa fa-shopping-cart'></i>
+    <span class="nav-icon-label">Cart</span>
+  <% end %>
+</li>
```

Comment:
why is the cart defined twice (here and _nav.erb), we may want to include a comment to ensure it is updated in both locations,  also what is the 'mini-cart' ID,where is it used?